protocols/rendezvous/src/server.rs

```diff
@@ -189,9 +189,14 @@ impl NetworkBehaviour for Behaviour {
                             handle_request(peer_id, request, &mut self.registrations)
                         {
                             if let Some(resp) = response {
-                                self.inner
+                                if let Err(resp) = self.inner
                                     .send_response(channel, resp)
-                                    .expect("Send response");
+                                    {
+                                        tracing::debug!(
+                                            %peer_id,
+                                            "Failed to send response: {resp:?}"
```

Comment:
```suggestion
                                            "Failed to send response, peer disconnected: {resp:?}"
```
we can assume that the peer disconnected right?

Reply:
yea, id say thats a safe assumption